osmosis-1/osmosis-1.assetlist.json

```diff
@@ -174,6 +174,37 @@
         "osmosis-main"
       ]
     },
+    {
+      "description": "PSTAKE Liquid-Staked ATOM",
+      "denom_units": [
+        {
+          "denom": "ibc/CAA179E40F0266B0B29FB5EAA288FB9212E628822265D4141EBD1C47C3CBFCBC",
+          "exponent": 0,
+          "aliases": ["stk/uatom"]
+        },
+        {
+          "denom": "stkatom",
+          "exponent": 6,
+          "aliases": ["stk/atom"]
+        }
+      ],
+      "base": "ibc/CAA179E40F0266B0B29FB5EAA288FB9212E628822265D4141EBD1C47C3CBFCBC",
+      "name": "PSTAKE staked ATOM",
+      "display": "stkatom",
+      "symbol": "stkATOM",
+      "ibc": {
+        "source_channel": "channel-6",
+        "dst_channel": "channel-4",
+        "source_denom": "stk/uatom"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/stkatom.png",
```

Comment:
Please instead point these image URIs to the Cosmos Chain Registry. This repo will no longer be storing images here.

Reply:
"png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkatom.png",
"svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkatom.svg"

Reply:
added, thanks!